dev-libs/benchmark/benchmark-1.3.0.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A microbenchmark support library"
+DESCRIPTION="A library to support the benchmarking of functions, similar to unit-tests."
+HOMEPAGE="https://github.com/google/benchmark"
+COPYRIGHT="2015-2017 Google Benchmark developers"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/google/benchmark/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="f19559475a592cbd5ac48b61f6b9cedf87f0b6775d1443de54cfe8f53940b28d"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
```

Comment:
The recipe is working fine on any primary arch, but on x86 secondary arch it fails to install because cmake is not told the correct path for the lib.
Hopefully this can be fixed. I'll try to see what we can do.

Reply:
I found a workaround: a `PATCH()` function, which can be inserted just before `BUILD()`. It will take care of fixing all paths used in `make install`.
```diff

	cmd:make
	"
+
+PATCH()
+{
+	sed -i \
+		-e "/^set(include_install_dir/ \
+			s|\"include\"|\"$relativeIncludeDir\"|;" \
+		src/CMakeLists.txt
+
+	if [ -n "$secondaryArchSuffix" ]; then
+		sed -i \
+			-e "/^set(lib_install_dir/ \
+				s|\"lib/\"|\"$relativeLibDir/\"|;" \
+			-e "/^set(config_install_dir/ \
+				s|\"lib/|\"$relativeLibDir/|;" \
+			src/CMakeLists.txt
+	fi
+}
+
 BUILD()
```
And in `INSTALL()` you should drop these 3 lines:
```diff
-	mkdir -p $includeDir
-	mv $prefix/include/* $includeDir/
-	rm -rf $prefix/include/
```

Reply:
I've tested your recipe (with my changes suggested above) and it is working fine on these architectures:
x86_64, x86 (both primary and secondary). Here are the tests results:
```
91% tests passed, 3 tests failed out of 34

The following tests FAILED:
	 29 - reporter_output_test (OTHER_FAULT)
	 31 - user_counters_test (OTHER_FAULT)
	 32 - user_counters_tabular_test (OTHER_FAULT)
```
(The results are the same on all architectures where it builds.)
This means you can mark x86 as tested:
```
ARCHITECTURES="!x86_gcc2 x86 x86_64"
SECONDARY_ARCHITECTURES="x86"
```

While at it, could you please also remove the old recipe, `benchmark-1.2.0.recipe` ?
Thanks!

Reply:
@DethroTull You should see if the tests could be fixed, if not please file a new issue concerning the tests failure.

Reply:
@fbrosson As you have tested the recipe, could you check what the soname version is? We should replace the `lib:libbenchmark = $portVersion` with [`libVersionCompat`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-libversioncompat-user-variable)